client/src/util/maybeScientific.js

```diff
@@ -0,0 +1,17 @@
+import significantDigits from "./significantDigits";
+
+export default function maybeScientific(x) {
```

Comment:
Please add a comment that describes the purpose of the function, the parameters and the return value.  For example:
```
/**
  * The function will ...
  * @param x - a d3 object which ....
  * @returns - ...
  */
```